manifests/init.pp

```diff
@@ -15,7 +15,10 @@
 # $version::                    foreman package version, it's passed to ensure parameter of package resource
 #                               can be set to specific version number, 'latest', 'present' etc.
 #
-# $port::                       Port on which will foreman proxy listen
+# $http_port::                  HTTP port to listen on (if http is enabled)
+#                               type:integer
+#
+# $ssl_port::                   SSL port to listen on (if ssl is enabled)
```

Comment:
I guess the other argument is it's https_port in the config file...but I like the convention in the parameters.